lib/src/mapper/mapper.dart

```diff
@@ -1,11 +1,4 @@
 import 'package:dolbyio_comms_sdk_flutter/dolbyio_comms_sdk_flutter.dart';
-import 'package:dolbyio_comms_sdk_flutter/src/sdk_api/models/conference.dart';
-import 'package:dolbyio_comms_sdk_flutter/src/sdk_api/models/events.dart';
-import 'package:dolbyio_comms_sdk_flutter/src/sdk_api/models/participant.dart';
-import 'package:dolbyio_comms_sdk_flutter/src/sdk_api/models/video_presentation.dart';
-
-import '../sdk_api/models/recording.dart';
-import '../sdk_api/models/streams.dart';
```

Comment:
As you will probably need this please investigate what is the preferred style in Dart,
* using absolute imports, ie. `import 'package:<path>.'`, or
* using relative imports ie. `import '../<path>'`.
There after please correct this as appropriate.